infra/gcp/tests/gke.tf

```diff
@@ -35,16 +35,99 @@ module "prow_trusted" {
       initial_node_count = 1
     },
   ]
+}
+
+module "prow" {
+  source                     = "terraform-google-modules/kubernetes-engine/google"
+  project_id                 = module.project.project_id
+  name                       = "prow"
+  region                      = "us-central1"
+  release_channel            = "RAPID"
+  network                    = "default"
+  subnetwork                 = "default"
+  ip_range_pods              = ""
+  ip_range_services          = ""
+  http_load_balancing        = true
+  network_policy             = false
+  horizontal_pod_autoscaling = true
+  filestore_csi_driver       = false
+  create_service_account     = false
+
+  node_pools = [
+    {
```

Comment:
Should there be a `zones` field here, based on our discussions in https://github.com/knative/test-infra/issues/3270?

Reply:
Fixed